server/sonar-web/src/main/webapp/WEB-INF/db/migrate/1452_make_uuid_not_null_on_events.rb

```diff
@@ -0,0 +1,30 @@
+#
+# SonarQube, open source software quality management tool.
+# Copyright (C) 2008-2016 SonarSource
+# mailto:contact AT sonarsource DOT com
+#
+# SonarQube is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 3 of the License, or (at your option) any later version.
+#
+# SonarQube is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program; if not, write to the Free Software Foundation,
+# Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+
+#
+# SonarQube 6.3
+#
+class MakeUuidNotNullOnEvents < ActiveRecord::Migration
+
+  def self.up
+    execute_java_migration('org.sonar.db.version.v63.MakeUuidNotNullOnEvents')
+    add_index 'events', :uuid, :name => 'events_uuid', :unique => true
```

Comment:
I thought there were already a Java component to add/remove index, but it's not the case, too bad.